impl/src/main/java/org/ehcache/loaderwriter/writebehind/BatchingLocalHeapWriteBehindQueue.java

```diff
@@ -152,9 +153,12 @@ private Batch newBatch() {
     return executor.submit(batch);
   }
   
+  @SuppressFBWarnings("IS2_INCONSISTENT_SYNC")
```

Comment:
Making the `openBatch` volatile makes this warning goes away - I believe that's the solution rather than the ignore.